mmv1/third_party/terraform/website/docs/r/composer_user_workloads_secret.html.markdown

```diff
@@ -0,0 +1,99 @@
+---
+subcategory: "Cloud Composer"
+description: |-
+  Kubernetes secret in Composer Environment workloads.
+---
+
+# google\_composer\_user\_workloads\_secret
+
+Provides a way to manage Kubernetes secret in Composer Environment workloads.
+
+## Example Usage
+
+```hcl
+resource "google_composer_environment" "example" {
+  name              = "example-environment"
+  project           = "example-project"
+  region            = "us-central1"
+  config {
+    software_config {
+      image_version = "example-image-version"
+    }
+  }
+}
+
+resource "google_composer_user_workloads_secret" "example" {
+  name = "example-secret"
+  project = "example-project"
+  region = "us-central1"
+  environment = google_composer_environment.example.name
+  data = {
+    email: base64encode("example-email"),
+    password: base64encode("example-password"),
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` -
+  (Required)
+  Name of the Secret.
+
+* `region` -
+  (Optional)
+  The location or Compute Engine region for the environment.
+
+* `project` -
+  (Optional)
+  The ID of the project in which the resource belongs.
+  If it is not provided, the provider project is used.
+
+* `environment` -
+  Environment where the secret will be stored and used.
+
+* `data` -
+  (Optional) 
+  The "data" field of Kubernetes Secret, organized in key-value pairs,
```

Comment:
btw, here we mention "Kubernetes Secret". Can you, please, make it consistent capital "Secret" in other places too